hack/components/bump_kubemacpool.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+
+KUBEMACPOOL_URL=$(cat components.yaml | shyaml get-value components.kubemacpool.url)
+KUBEMACPOOL_COMMIT=$(cat components.yaml | shyaml get-value components.kubemacpool.commit)
+KUBEMACPOOL_REPO=$(echo ${KUBEMACPOOL_URL} | sed 's#https://\(.*\)#\1#')
+KUBEMACPOOL_PATH=${GOPATH}/src/${KUBEMACPOOL_REPO}
+
+echo 'Fetch kubemacpool sources'
+fetch_component ${KUBEMACPOOL_PATH} ${KUBEMACPOOL_URL} ${KUBEMACPOOL_COMMIT}
+
+echo 'Configure kustomize for CNAO templates and save the rendered manifest under CNAO data'
+(
+    cd ${KUBEMACPOOL_PATH}
+    mkdir -p config/cnao
+
+    cat <<EOF > config/cnao/kustomization.yaml
```

Comment:
nit:
just if you want, you can use ident, with tab only, if there arent variables i think
cat <<-EOF
    indented
    EOF
echo Done

Reply:
sounds nice, but I tried but it cancels indents of internal lines, for example:
if I have
```
cat <<-EOF
(tab)X1:
(tab)(tab)X1.1: value
(tab)X2:
(tab)EOF
```

it flatten all tabs to:
```
X1:
X1.1= value
X2:
```

Reply:
(tab)EOF
i think shouldnt be with tab ?
https://unix.stackexchange.com/questions/353076/how-to-indent-an-heredoc-inside-an-heredoc-the-right-way?noredirect=1&lq=1

Reply:
From what I saw EOF should also be indented if we use <<-EOF